app/views/support_interface/change_course/select_choice_to_withdraw.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :before_content, govuk_back_link_to(support_interface_change_course_path(@form.application_form)) %>
```

Comment:
There isn't a title on this page; which should contain `Error:` like the above suggestion.